fixtures/redirects_redirect.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
@martinmacko47 Funguje ti redirect? Postupoval som podla navodu. Najprv mi to fungovalo, ale teraz mi to nejde. Teda najpr som si vytvoril iba skusobny redirect `"old_path": "/tmp/", ktory fungoval. Po vymazani tohto redirectu z databazy sa zmena neprejavila a stale ked zadam do browsera `"127.0.0.1:8000/tmp/"` tak ma redirecte na `new_path`. A zaroven ked pridam dalsi zaznam do databazy, teda novy redirect napr. `"/foo/"` tak vyhodi chybu 404.

Ad2: Ako to treba osetrit s roznymi jazykmi. Funguje mi redirect `/en/tmp`, `/tmp/` ale nefunguje mi `/sk/tmp/`. Poslal by si mi prosim ten vzorovypriklad ako si spominal v popise issue? dakujem

Reply:
> Po vymazani tohto redirectu z databazy sa zmena neprejavila a stale ma redirecte

To sa moze stat, ak sa pouzviva [301 Moved Permanently](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301). Takze browser si ten redirect zapamata a nabuduce redirectne hned bez volania servera. Take najlepsie testovat cez private mod. Aby po zavreti okna s private modom browser zabudol vsetky nacachovane veci.

> A zaroven ked pridam dalsi zaznam do databazy tak vyhodi chybu 404

To zneje divne. Tak by sa to nemalo spravat. Nezabudol si restartnut server alebo nemal si nejaky rozdiel v URL? Napriklad `/en` vs `/sk` prefix?

> Ako to treba osetrit s roznymi jazykmi

Kazda site by mala mat vlastnu sadu redirectov. Kedze kazda site ma vlastne URL. V tom druhom projekte mame iba jednu site, takze tam to nemame poriesene. Ale v principe treba spravit asi viac verzii `redirects_redirect.json`, jednu pre kazdu site. A potom v setupe v `load_redirects` ich naloadovat pre prislusne sites. Linky na vzorovy priklad vid v slacku.

Reply:
:ok: Zatial som nechal iba jeden subor, kedze zatial jediny redirect je `/sk/`

Reply:
@viliambalaz Ok. Moze byt len jeden subor zatial.